ui/src/dashboards/components/DisplayOptions.js

```diff
@@ -2,6 +2,9 @@ import React, {Component, PropTypes} from 'react'
 
 import GraphTypeSelector from 'src/dashboards/components/GraphTypeSelector'
 import AxesOptions from 'src/dashboards/components/AxesOptions'
+import SourceSelector from 'src/dashboards/components/SourceSelector'
+
+import _ from 'lodash'
```

Comment:
extremely minor, but generally we've been putting these libraries just under the React import itself, before component imports